client/src/Main/Main.test.js

```diff
@@ -8,6 +8,7 @@ const publicRoutes = [
   '/projects',
   '/register',
   '/login',
+  '/contact/:id'
```

Comment:
This is making the test suite fail. The array should only contain public routes, the private routes are created dynamically so cannot be tested by `shallow`. There's a sub-section of tests further down for private routes.